_i18n/en/InternMountainDay.md

```diff
@@ -0,0 +1,10 @@
+On December, 11th I was invited to celebrate International Mountain Day and speak on behalf of the project. Together with other PhD students we discussed both professionally but also personally how mountains matter to us and consequently how our research related to our personal understanding of the topic.

```

Comment:
You also have to add the header here.

Please follow exactly the template here: https://github.com/agroclim-huaraz/agroclim-huaraz-website/tree/master/_i18n/en/_posts (also the filename is really important, otherwise jekyll doesn't know what to do with the posts)